Jenkinsfile

```diff
@@ -137,7 +137,6 @@ try {
               sh "./scripts/test/hyriseConsole_test.py gcc-debug"
               sh "./scripts/test/hyriseBenchmarkJoinOrder_test.py gcc-debug"
               sh "./scripts/test/hyriseBenchmarkFileBased_test.py gcc-debug"
-              sh "./scripts/test/hyriseBenchmarkTPCC_test.py gcc-debug"
```

Comment:
Reason?

Reply:
This whole step is the slowest one, so we rather execute TPC-C in a different step